src/nycdb/dataset.py

```diff
@@ -169,7 +170,48 @@ def verify(self):
         if it contains approximately the right number of rows.
         """
         self.setup_db()
-        verify.check_dataset(self.db, self.name)
+
+        exit_state = True
+
+        for schema in self.schemas:
+            table_name = schema["table_name"]
+            if self.db.table_exists(table_name):
```

Comment:
to reduce nesting and make this section a little easier to read, what do you think of inverting this first condition and doing this:
```
if not self.db.table_exists(table_name):
  exit_state = False
  print(colorize("fail", table_name + " is missing!"))
  continue
...
```